src/logic/LocalMessageCache.tsx

```diff
@@ -132,6 +137,23 @@ class LocalMessageCache {
         return;
     }
 
+    static all(
+        slotName: SlotName,
+        appboyInstance: typeof appboy,
+        errorHandler: ErrorHandler,
+    ): MessageWithId[] {
+        const queue = getQueue(slotName, errorHandler);
+
+        if (queue) {
```

Comment:
No harm, but looking at the way `peek` uses `getQueue` and the return type of `getQueue` I'm not sure this condition is necessary. It looks like we'll always get an array of type `CachedMessage[]` back so we can probably always just return `queue.map(...)` here. What do you reckon?

Reply:
Fixed in https://github.com/guardian/braze-components/pull/151/commits/ec8a83027ff6992bcef55ef2b9f874c5a3b02e26